README.md

```diff
@@ -96,7 +124,15 @@ Note that you can build the node in release mode this way:
 catkin build -w ~/catkin_ws --cmake-args -DCMAKE_BUILD_TYPE=Release
 ```
 
-9. Clone the PX4 Firmware and all its submodules (it may take some time).
+10. Source the catkin setup.bash from your catkin workspace.
+
+```bash
+source ~/catkin_ws/devel/setup.bash
+```
+
+## Run the Simulation
```

Comment:
Run what simulation, on what simulator?

Before starting on the steps, perhaps a few sentences stating *what* you demonstrate here. Ie what simulator (Gazebo), running on what OS (Ubuntu 16.04), demonstrating what  what planners (local and global planner), meeting what use cases (collision prevention? Obstacle avoidance in missions?). 

You might capture some small subset of that in the heading itself. Depends on what you are showing.

Reply:
Also, what has to be setup already to get to this point? Do you already need ROS setup?